etcdserver/api/v3rpc/lease.go

```diff
@@ -73,6 +73,17 @@ func (ls *LeaseServer) LeaseKeepAlive(stream pb.Lease_LeaseKeepAliveServer) erro
 		resp := &pb.LeaseKeepAliveResponse{ID: req.ID, Header: &pb.ResponseHeader{}}
 		ls.hdr.fill(resp.Header)
 
+		if req.Probe {
```

Comment:
have methods `LeaseServer.probe(id) (*LeaseKeepAliveResponse, error)` and `LeaseServer.renew(id) (*LeaseKeepAliveReponse, error)` instead of handling both directly in this function?